air/src/options.rs

```diff
@@ -265,8 +265,8 @@ impl ExecutionOptions {
     /// - Executes `debug` instructions (these are ignored in regular mode).
     /// - Records additional info about program execution (e.g., keeps track of stack state at every
     ///   cycle of the VM) which enables stepping through the program forward and backward.
-    pub fn with_debugging(mut self) -> Self {
-        self.enable_debugging = true;
+    pub fn with_debugging(mut self, enable_debugging: bool) -> Self {
+        self.enable_debugging = enable_debugging;
```

Comment:
nit: could we add a short comment on the the usage of the flag `enable_debugging``?

Reply:
done